packages/xfsprogs/0001-libxfs-do-not-try-to-run-the-crc32selftest.patch

```diff
@@ -0,0 +1,44 @@
+From 353e3b5a88849efced5b9faedbf8b64c6a183933 Mon Sep 17 00:00:00 2001
+From: Matthew Yeazel <yeazelm@amazon.com>
+Date: Tue, 13 Jun 2023 20:06:06 +0000
+Subject: [PATCH] [PATCH] libxfs: do not try to run the crc32selftest
```

Comment:
Let's keep the original buildroot patch header for attribution purposes.

Reply:
Whoops, I meant to do this and spaced it, thanks for the catch!